database/flooddev/u_flood/4.3.0/stations_list_mview.sql

```diff
@@ -1,84 +1,109 @@
 -- View: u_flood.stations_list_mview
```

Comment:
A lot of the diffs here are due to the autogenerated scripts from pgadmin not things i think should change, happy to tidy them up to make the diff clearer (only real change is to add the qualified name).

Reply:
I've used changed this so the diff is (hopefully) cleaner.